readme.md

```diff
@@ -210,12 +210,15 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 ## Changelog
 
 ### 3.0.0 (XX XXX 2021)
-- The minimum version of Jenkins changed to be **2.204.4**
-- A number of dependencies upgrades
+- The minimum version of Jenkins changed to be **2.204.6**
+- JENKINS-60342 added support for Pull Request triggers
+- JENKINS-63033 added support for lightweight checkout with pipeline and mutlibranch pipeline jobs
```

Comment:
Should be "multibranch".

Reply:
:facepalm: I've made that typo so many times. Thanks for catching